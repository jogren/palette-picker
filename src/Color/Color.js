import React from 'react';

const Color = ({style}) => {
  return (
    <article style={style}>
      <button>lock</button>
    </article>
  )
}

export default Color;