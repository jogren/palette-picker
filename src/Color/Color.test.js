import React from 'react';
import { shallow } from 'enzyme';
import Color from './Color';

describe('Color', () => {
  let wrapper;
  const toggleLockMock = jest.fn();
  const colorMock = { 
    backgroundColor: "#FFFFFF"
  };

  beforeEach(() => {
    wrapper = shallow(<Color 
      color={colorMock}
      toggleLock={toggleLockMock} />)
  });

  it('should match the snapshot with the data passed through', () => {
    expect(wrapper).toMatchSnapshot();
  });
});