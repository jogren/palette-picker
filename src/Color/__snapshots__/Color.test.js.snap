// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Color should match the snapshot with the data passed through 1`] = `
<article
  className="color"
  style={
    Object {
      "backgroundColor": "#FFFFFF",
    }
  }
>
  <button
    onClick={[Function]}
  >
    <FaLockOpen />
  </button>
  <p>
    #FFFFFF
  </p>
</article>
`;
